Project-Aurora/Project-Aurora/Profiles/Stardew Valley/GSI/Nodes/PlayerNode.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Aurora.Profiles.StardewValley.GSI.Nodes
+{
+
+    public class PlayerNode : AutoJsonNode<PlayerNode>
+    {
+        public HealthNode Health => NodeFor<HealthNode>("Health");
+        public string CurrentLocation;
```

Comment:
is there no Enum for locations?

Reply:
Nope. But I could look at the locations and create my own enum.